src/test/java/hook/SpaceHook.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
+package hook;
+
+import clickup.ui.entities.Context;
+import clickup.ui.pages.SpacePage;
+import cucumber.api.java.After;
+
+/**
+ * Hook class.
+ *
+ * @author Jesus Menacho
+ * @version 1.0
+ */
+public class SpaceHook {
+    private Context context;
+
+    /**
+     * Constructor initializes the class setting the context.
```

Comment:
Please change the comment according to our Code Conventions and add an empty line.

Reply:
Done.